### DISCUSSION TITLE
Usefetch not working

### DISCUSSION THREAD

**Discussion by dev-param** [nuxt/nuxt#21580] [Questions]
When I turn off SSR : false usefetch not working 
And not showing any error

But work with SSR: true

**Comment by warflash** [comment]
Can you add a reproduction?

**Comment by dev-param** [comment]
What is reproduction

**Reply by moebiusmania** [reply]
it means if you have some repositeries or code snippet in general that everyone could try locally on their machine and reproduce your issue to try to analyze the problem.

**Comment by dev-param** [comment]
https://github.com/nuxt/nuxt/discussions/21580#discussion-5301096

app.vue->components->home.vue
`const { data, pending, error, refresh } = await useFetch("http://127.0.0.1:8000", {server: false})`

 ```
 "devDependencies": {
    "@types/node": "^18",
    "autoprefixer": "^10.4.14",
    "nuxt": "^3.5.2",
    "postcss": "^8.4.24",
    "tailwindcss": "^3.3.2"
  },
  "dependencies": {
    "@heroicons/vue": "^2.0.18"
  }
```



// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
  
    css: ['~/assets/css/main.css'],

    postcss: {
        plugins: {
          tailwindcss: {},
          autoprefixer: {},
        },
      },
      
    
})
